wgsl/index.bs

```diff
@@ -8769,6 +8692,32 @@ WebGPU provides no guarantees about:
     the invocations of another workgroup.
     That is, you cannot assume that workgroups are launched in a particular order.
 
+## Fragment Shaders and Helper Invocations ## {#fragment-shaders-helper-invocations}
+
+Invocations in the [=fragment shader stage=] are divided into 2x2 grids of neighbouring invocations.
+Each of these grids is referred to as a <dfn noexport>quad</dfn>.
+Quads can collaborate in some collective operations (see [[#derivatives]]).
+
+An implementation will always ensure that a shader is executed with enough
+invocations such that each fragment is part of a full quad.
+When there are not enough fragments to fill the quad, an implementation will
```

Comment:
It's a little wobbly to say the implementation will make fragments, then say if there aren't enough fragments.

I'd tighten this up by referring back to the API spec, specifically https://gpuweb.github.io/gpuweb/#rasterizationpoint 


> Ordinarily, [fragment processing](https://gpuweb.github.io/gpuweb/#fragment-processing) creates one invocation of a fragment shader for each [RasterizationPoint](https://gpuweb.github.io/gpuweb/#rasterizationpoint) produced by [rasterization](https://gpuweb.github.io/gpuweb/#rasterization). 
Sometimes there may be  insufficient RasterizationPoints to fully populate a quad, for example at the edge of a graphics primitive. 
When a quad has 1, 2, or 3 invocations corresponding Rasterization points, fragment processing will create a `<dfn noexport>`helper invocation`<dfn>` for each unpopulated position in the quad.